packages/flutter_tools/lib/src/commands/init.dart

```diff
@@ -201,14 +201,27 @@ void main() {
   runApp(
     new MaterialApp(
       title: "Flutter Demo",
-      routes: {
+      routes: <String, RouteBuilder>{
         '/': (RouteArguments args) => new FlutterDemo()
       }
     )
   );
 }
 
-class FlutterDemo extends StatelessComponent {
+class FlutterDemo extends StatefulComponent {
+  @override
```

Comment:
We've been skipped it in the framework because there isn't great tooling support for it.